$MirOS: ports/games/bsdgames/patches/patch-rogue_use_c,v 1.1.7.1 2005/03/18 15:46:07 tg Exp $
--- rogue/use.c.orig	Thu Jan  1 00:00:00 1970
+++ rogue/use.c	Thu Feb 22 21:51:08 2007
@@ -367,7 +367,7 @@ AGAIN:
 	}
 	if (!(obj = get_letter_object(ch))) {
 		messagef(0, "no such item, try again");
-		messagef(0, "");
+		messagef(0, NULL);
 		check_message();
 		goto AGAIN;
 	}
